db.go

```diff
@@ -856,7 +856,7 @@ func (db *DB) buildOtherIdxes(r *Record) error {
 	switch r.H.Meta.Ds {
 	case DataStructureList:
 		if err := db.buildListIdx(r); err != nil {
-			return err
+			return fmt.Errorf("when build listIdx err: %s", err)
```

Comment:
The string can be defined into a const string variable.  That will make the code more elegant.

Reply:
> The string can be defined into a const string variable. That will make the code more elegant.

OK : )

Reply:
> The string can be defined into a const string variable. That will make the code more elegant.

I refactored code so that `fmt.Errorf` was only called once and I didn't use `const` to define this pattern string, because I think it would be more intuitive if only called once.